index.js

```diff
@@ -7,10 +7,17 @@ const path = '/siteverify';
 // verifies the given token by doing an HTTP POST request
 // to the hcaptcha.com/siteverify endpoint by passing the
 // hCaptcha secret key and token as the payload.
-const verify = (secret, token) => {
+const verify = (secret, token, remoteip, sitekey) => {
```

Comment:
We could add defaults the optional parameters (i.e. `remoteip = null`) and simplify the payload to:

```js
const payload = {secret, remoteip, sitekey, response: token};
```

Reply:
but then we send empty params in the request to hcaptcha while their documentation states they should be optional, they should already be optional in the typescript interface

```
  export function verify(secret: string, token: string, remoteip?: string, sitekey?: string): Promise<VerifyResponse>;
```

i'll anyway add the null assignment to this method signature.